openedx/features/enterprise_support/context.py

```diff
@@ -0,0 +1,34 @@
+"""
+APIs providing support for enterprise functionality.
```

Comment:
@rehanedly  need to change this to `APIs providing enterprise context for events`

Reply:
@ziafazal done